### ISSUE TITLE
Update the Design System

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2684]
### Overview
We need to audit all the latest Figma screens to find out what new elements need to be added to the design system. 

### Details
- There are many new Figma design elements that are not using the design system. 
- Once this issue is complete we will create a separate issue to update the design system with these findings.

### Action Items
- [ ] Define the approach you will take (write in a comment below)
- [ ] Discuss with Design lead
- [ ] Confirm with PM once the scope and outcome of plan to defined, so that you can be sure that we are in alignment

### Resources/Instructions
(Provide links to resources or instructions that may help with this issue. This can include files to be worked on, external sites with solutions, documentation, etc.)

- [Figma - Current Pages](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A0)
